command/polybftmanifest/manifest_init.go

```diff
@@ -0,0 +1,193 @@
+package polybftmanifest
+
+import (
+	"bytes"
+	"fmt"
+	"path"
+	"strings"
+
+	"github.com/0xPolygon/polygon-edge/command"
+	"github.com/0xPolygon/polygon-edge/command/genesis"
+	"github.com/0xPolygon/polygon-edge/consensus/polybft"
+	"github.com/0xPolygon/polygon-edge/types"
+	"github.com/spf13/cobra"
+)
+
+const (
+	manifestPathFlag      = "path"
+	premineValidatorsFlag = "premine-validators"
+	validatorsFlag        = "validators"
+	validatorsPathFlag    = "validators-path"
+	validatorsPrefixFlag  = "validators-prefix"
+
+	defaultValidatorPrefixPath = "test-chain-"
+	defaultManifestPath        = "./manifest.json"
+
+	nodeIDLength       = 53
+	ecdsaAddressLength = 42
+	blsKeyLength       = 2
+)
+
+var (
+	params = &manifestInitParams{}
+)
+
+// GetCommand returns the rootchain emit command
+func GetCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "polybft-manifest",
+		Short: "Initializes manifest file",
+		// PreRunE: runPreRun,
```

Comment:
Remove this since we don't have a pre-run.

Reply:
Now we have it :stuck_out_tongue: 
https://github.com/0xPolygon/polygon-edge/pull/1005/commits/8a965c3c489384f6e32ea9ace2338dd3b7ceb997